inscriptions/ISic0980.xml

```diff
@@ -171,38 +176,23 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140457">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
                 <ab>
-                    <lb n="1"/>
-
-
-                    ἐνθάδε κῖτε<lb n="2"/>Pαῦλα <surplus>²⁶Παῦλα</surplus>²⁶, ζή<supplied reason="omitted">σ</supplied>ασ
-                    <supplied reason="lost">α</supplied>
-                    <lb n="3"/>ἔτη κεʹ· <supplied reason="omitted">ἀ</supplied>νέλ
-                    <supplied reason="omitted">υσ</supplied>
-                    <supplied reason="lost">ε</supplied>
-                    <lb n="4"/>δὲ τὸν bίον <surplus>²⁶βίον</surplus>²⁶ πρὸ ηʹ καλ<supplied reason="lost">αν</supplied>-
-                    <lb n="5"/>δῶν Δεκενβρίων ∙
-                    <supplied reason="omitted">ὑ</supplied>
-                    <expan>
-                        <abbr>πα</abbr>
-                        <ex>τίᾳ</ex>
-                    </expan>
-                    <lb n="6"/>Ἱερίου καὶ Ἀρ
-                    <supplied reason="omitted">δαβ</supplied>
-                    <expan>
-                        <abbr>ουρί</abbr>
-                        <ex>ου</ex>
-                    </expan>
-                    τ
-                    <supplied reason="omitted">ῶ</supplied>
-                    <supplied reason="lost">ν</supplied>
-                    <lb n="7"/>λαμπροτάτων.
-                    <lb n="8"/>
+                    <lb n="1"/>ἐνθάδε κῖτε
+                    <lb n="2"/><choice><corr>Π</corr><sic><foreign xml:lang="la">Ρ</foreign></sic></choice>αῦλα ζή<choice><corr>σ</corr><sic>ι</sic></choice>α<unclear>σ</unclear><supplied reason="omitted">α</supplied>
+                    <lb n="3"/>ἔτη <num value="25">κε</num>· <choice><corr>ἀ</corr><sic>λ</sic></choice>νέλ<choice><corr>υσε</corr><sic><orig>ΙΙ</orig></sic></choice>
+                    <lb n="4"/>δὲ τὸν <choice><corr>β</corr><sic><foreign xml:lang="la">b</foreign></sic></choice>ίον πρὸ <num value="8">η</num> καλ<supplied reason="lost">αν</supplied>
+                    <lb n="5" break="no"/>δῶν Δεκενβρίων <g type="interpunct">·</g> <surplus>τ</surplus> <expan><abbr><supplied reason="lost">ὑ</supplied>πα</abbr><ex>τίᾳ</ex></expan>
+                    <lb n="6"/>Ἱερίου καὶ Ἀρ<choice><corr>δαβ</corr><sic><orig>ΙΟΙΙ</orig></sic></choice>ουρί<supplied reason="omitted">ου</supplied> τ<choice><corr>ῶ</corr><sic>η</sic></choice><supplied reason="lost">ν</supplied>
```

Comment:
In the orig, is this a 0 (zero) or O (capital o)? Looks a bit squished like a zero here but it might just be the github interface.